go/runtime/txpool/local_queue.go

```diff
@@ -69,6 +69,12 @@ func (lq *localQueue) HandleTxsUsed(hashes []hash.Hash) {
 	lq.txs = keptTxs
 }
 
+func (lq *localQueue) PeekAll() []*TxQueueMeta {
+	lq.l.Lock()
+	defer lq.l.Unlock()
+	return append(make([]*TxQueueMeta, 0, len(lq.txs)), lq.txs...)
```

Comment:
I prefer 3 lines. Should we preallocate also in `GetTxsToPublish`?

Reply:
I put this on a single line to better fit with the rest of the coding style in the txpool package (a few other existing methods also do this).